### ISSUE TITLE
AsPartitionProcessing

### ISSUE THREAD

**Issue by dwul** [microsoft/Analysis-Services#94]
Hi there,

I am facing problem with the AsPartitionProcessing solution.

We are using this framework to partition an SSAS model (PBI dataset) into daily and monthly partitions.

So far, everything works fine. When moving from one month to the other, all daily partitions are deleted and a new daily partition for the first day of the month is created. Unfortunately, the deleted partitions of the previous month are not stored in a monthly partitions so we basically lose that data.
To sum up, when there is a new month, no new monthly partition is created (e.g. on the 1st of February we only have one daily partition covering the 1st of February and there is no monthly partition created for January) . Right now, we need to fix that problem by manually processing the whole SSAS table all over again.

Any help appreciated! Thanks!
dwul

**Comment by Clapton79** [comment]
You can merge a day into another partition . So jan days could go into the
partition for 1st jan. As per the feb one you can create empty partitions
with xmla prior to a given day

**Comment by byobi** [comment]
You'll want to manage this through the partition configuration table. 

Take a look at the example from the documentation dealing with mixed granularities. Basically, you will need an (automated) process to create/update/delete the rows in the partition configuration table for any target tables that require mixed granularities.

**Comment by dwul** [comment]
We are still facing issues here.

These are our ConfigurationTables:

Table Configuration
![TableConfiguration](https://user-images.githubusercontent.com/79318654/109946434-84ef7d80-7cd8-11eb-8bbb-351067e154c3.PNG)

Model Configuration
![ModelConfiguration](https://user-images.githubusercontent.com/79318654/109946493-96388a00-7cd8-11eb-906f-262b2eeed809.PNG)

PartitioningConfiguration
![PartitioningConfiguration](https://user-images.githubusercontent.com/79318654/109946529-9f295b80-7cd8-11eb-977b-1097d8e627b8.PNG)

and finally the logging where no monthly partition was created. This totally working fine on the daily level. The only difference in the logging table is that on monthly level no new partition is created.

![Logging](https://user-images.githubusercontent.com/79318654/109946747-de57ac80-7cd8-11eb-8053-714d4a46580d.PNG)

Some additional information:
There are two tables: Price and Date (hours)
Price is supposed to be partitioned (daily partitions for the current month, monthly partitions for the last 12 months, delete everything that is older than 12 months)
Date (hours) is not meant to be partitioned (this is working fine).

Thanks for any support here :)
